### ISSUE TITLE
num_workers=1

### ISSUE THREAD

**Issue by wangfugui127** [xuebinqin/U-2-Net#381]
salobj_dataloader = DataLoader(salobj_dataset, batch_size=batch_size_train, shuffle=True, num_workers=1)
 num_workers=1 导致我的程序无法正常运行，是我的电脑无法多个进程吗
